js/compileShell.py

```diff
@@ -35,19 +35,19 @@
 S3_SHELL_CACHE_DIRNAME = 'shell-cache'  # Used by autoBisect
 
 if sps.isWin:
-    MAKE_BINARY = 'mozmake'
-    CLANG_PARAMS = ' -fallback'
-    # CLANG_ASAN_PARAMS = ' -fsanitize=address -Dxmalloc=myxmalloc'
+    MAKE_BINARY = str("mozmake")
```

Comment:
This will be wrong again on py3 + windows.
All of the `"..."` ➡️ `str("...")` changes should instead be `"..."` ➡️ `b"..."`